CHANGELOG.md

```
@@ -1,5 +1,136 @@
+
+- Add a link to github workflow badges by @Olexandr88 in [#379](https://github.com/holochain/holochain-client-js/pull/379)
+
+### Other Changes
+
+- Build/docs error (#391) by @mattyg in [#391](https://github.com/holochain/holochain-client-js/pull/391)
+- Feat/missing types from holochain open dev core types (#388) by @mattyg in [#388](https://github.com/holochain/holochain-client-js/pull/388)
+
+### First-time Contributors
+
+- @Olexandr88 made their first contribution in [#379](https://github.com/holochain/holochain-client-js/pull/379)
+# Changelog
```

markdownlint-cli2 (0.23.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)